portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,20 +36,33 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+
     
   private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
+  private final ArrayList<String> comments = new ArrayList<>();
+
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    System.out.println("hi");
     Query query = new Query("Comment").addSort("year", SortDirection.DESCENDING)
                                       .addSort("month", SortDirection.DESCENDING)
                                       .addSort("day", SortDirection.DESCENDING);
 
     PreparedQuery results = datastore.prepare(query);
 
+    int commentAmount = 10;
+    String qString = request.getParameter("comment-amount");
```

Comment:
qString seems a bit generic. It's always ideal to be able to tell a variable's purpose from its name. E.g. maxPurchaseAmount vs maxAmt or maxAmount. Sometimes this means longer variable names as a cost, but is usually a better investment in the long-run. Can we think of a more descriptive variable name here?

Reply:
I'll change it to amountParam I had it as qString because it was originally set to request.getQueryString() and I just hadn't changed it.